pages/dkp/kommander/2.2/install/air-gapped/index.md

```diff
@@ -175,14 +175,26 @@ export VERSION=v2.2.0
     wget "https://downloads.mesosphere.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar.gz" -O kommander-image-bundle.tar.gz
     ```
 
-1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+1.  Download the [DKP catalog applications][dkp_catalog_applications] image bundle if you intend on deploying any DKP catalog applications:
+
```

Comment:
I guess this is an optional step? Some customers don't need/can't use catalog apps

Reply:
overlooked the `if` ... nvm